testsuite/run_sets/build_validation_add_none_MU_channels.yml

```diff
@@ -0,0 +1,13 @@
+## Add none_MU repositories BEGIN ###
+
+- features/build_validation/add_none_MU_repositories/ubuntu1804_minion_add_none_MU_repositories.feature
+- features/build_validation/add_none_MU_repositories/alma9_minion_build_clm.feature
```

Comment:
Strictly speaking, the CLM projects are for both `minion` and `SSH minion`. But that's OK with `minion` in the name.